packages/node/tsconfig.test.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Do we need to have two of these?

Reply:
The only difference here is `"include": ["src", "test"],` which allows for compiling the tests, but not rolling up tests (since the `tsconfig.json` is used for rollup)

Reply:
Seems like the rolls should be reversed then, e.g., `tsconfig.rollup.json` should be the edge case and `tsconfig.json` be the one that includes `src` and `test`

Reply:
That works too. Could either be `tsconfig.test.json` or `tsconfig.rollup.json`

I prefer your suggestion actually.